docs/Documentation/Systems/Swift/running.md

```diff
@@ -319,13 +322,7 @@ module load intel-oneapi-mpi intel-oneapi-compilers gcc
 
 ```
 
-Launch with the srun command:
-
-```bash
-srun   ./a.out -F
-```
-
-Our IntelMPI batch script is:
+Our IntelMPI batch script, *runintel* under */example*, is (remember to replace `<myaccount>` with the appropriate account):
```

Comment:
I'm trying to move towards highlighting file paths in code highlighting rather than italics. So "`" instead of "*".

Reply:
should I change all *path* to 'path' in the code?